docs/01-Using-Fleet/standard-query-library/standard-query-library.yml

```diff
@@ -918,4 +918,89 @@ spec:
   platforms: macOS
   tags: compliance, CIS, CIS6.1.4
   platform: darwin
-  contributors: GuillaumeRoss
\ No newline at end of file
+  contributors: GuillaumeRoss
+
+---
+    apiVersion: v1
+    kind: policy
+    spec:
+      name: Ensure Windows Firewall - Domain Profile is enabled (Windows)
+      query: SELECT 1 FROM registry WHERE path LIKE 'HKEY_LOCAL_MACHINE\Software\Policies\Microsoft\WindowsFirewall\DomainProfile\EnableFirewall' AND CAST(data as integer) = 1;
+      description: "Checks for the existence of a GPO that requires the Windows Firewall - Domain Profile is enabled."
```

Comment:
As discussed - I'd replace GPO with "policy" pretty much everywhere in these queries!